### ISSUE TITLE
Tests run and pass, but task is an overall "failure" because Pester couldn't download from powershellgallery.com.

### ISSUE THREAD

**Issue by BigScary** [rfennell/AzurePipelines#677]
Our task is failing with "Package 'Pester' failed to download.", but tests execute and pass.  If tests are able to execute and pass, then the failure to download should be reported as a warning rather than a failure, so subsequent pipeline steps can proceed.

2020-04-08T04:39:47.4316922Z Preparing task execution handler.
2020-04-08T04:39:47.7911448Z Executing the powershell script: E:\_work\_tasks\Pester_31ef0033-64e3-4c55-b888-f446541474a6\8.13.3\Pester.ps1
2020-04-08T04:39:47.9882122Z Running in AMD64 PowerShell
2020-04-08T04:39:47.9882441Z 
2020-04-08T04:39:47.9882632Z 
2020-04-08T04:39:47.9919006Z Adding additional module path [] to $env:PSModulePath
2020-04-08T04:39:47.9919321Z 
2020-04-08T04:39:47.9919470Z 
2020-04-08T04:41:25.3518522Z ##[warning]Source Location 'https://www.powershellgallery.com/api/v2/package/Pester/4.10.1' is not valid.
**2020-04-08T04:41:25.3580782Z ##[error]System.Exception: Package 'Pester' failed to download.**
2020-04-08T04:41:28.6655107Z Running Pester from using the script parameter [@{Path = REDACTED ; Parameters = @{ logFolder = REDACTED } }] output sent to [REDACTED]
2020-04-08T04:41:28.6655594Z 
2020-04-08T04:41:28.6655695Z 
2020-04-08T04:41:29.8990742Z     ____            __
2020-04-08T04:41:29.8991083Z    / __ \___  _____/ /____  _____
2020-04-08T04:41:29.8991244Z   / /_/ / _ \/ ___/ __/ _ \/ ___/
2020-04-08T04:41:29.8991391Z  / ____/  __(__  ) /_/  __/ /
2020-04-08T04:41:29.8991535Z /_/    \___/____/\__/\___/_/
2020-04-08T04:41:29.8991655Z Pester v4.9.0
2020-04-08T04:41:29.8991861Z Executing all tests in 'REDACTED'

### Azure DevOps Extension you are using
8.13.3

### Where are you running it?
*You can run the extensions in the cloud or on-premise*
- Azure DevOps Service (VSTS)

### Version of Extension/Task
Latest public version.

**Comment by rfennell** [comment]
@ChrisLGardner any thoughts?

**Comment by ChrisLGardner** [comment]
It's probably related to some TLS changes the team made recently, but should be reverted I believe (at least for now).

I need to make a change to set the max version to download as 4.99.99 or something as 5 is coming out soon and will have breaking changes.

I'll try to get that done today or tomorrow and add some extra checks in for this and handling the errors too. It'll be the last proper release of this extension before we deprecate it and move to the official one, which I should have time to work on this week.

**Comment by rfennell** [comment]
@ChrisLGardner I think we might have to release that one manually as we are going to have version number increment issue. 

Also if you are making a change to the version in this repo can you change the extension manifest so that all Q&A goes to Github - I missed that some questions had been raise din the Marketplace so was late to respond 

```
 "CustomerQnASupport": {
    "enableqna":"true",
    "url": "https://github.com/rfennell/AzurePipelines/issues"
  }, 
```

**Comment by ChrisLGardner** [comment]
Will do.

**Comment by ChrisLGardner** [comment]
This should be fixed now with release 8.14.5

**Comment by ChrisLGardner** [comment]
Closing as this should be resolved now.
